test/pages/AreThereRestrictionsOnTheGoodsPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class AreThereRestrictionsOnTheGoodsPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](AreThereRestrictionsOnTheGoodsPage)
   }
+  "cleanup" - {
+    "should reset DescribeTheRestrictionsPage" - {
+      "when AreThereRestrictionsOnTheGoodsPage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(DescribeTheRestrictionsPage, "restrictions")
+          .get
+
+        AreThereRestrictionsOnTheGoodsPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
+  }
```

Comment:
We want to be testing all  scenarios here, namely what happens when the cleanup is called with a `None`, and `Some(true)`.

Reply:
Good call, adding now